lib/fastlane/plugin/appcenter/version.rb

```diff
@@ -1,5 +1,5 @@
 module Fastlane
   module Appcenter
-    VERSION = "1.4.0"
+    VERSION = "1.5.0"
```

Comment:
Do I bump this now? I can remove this commit if needed

Reply:
We will bump it as part of the release process.

Reply:
👍 I'll remove this